scripts/portstat

```diff
@@ -66,7 +66,6 @@ filter_bucket_dict = {
     'RFCR': 1,
     'RJBR': 1,
     'RDBGC0': 2,
-    'RDBGC2': 2,
```

Comment:
This was a change I merged from our internal repo. You removed this, claiming it was an unnecessary drop counter on 5/12/2016.